cli.mjs

```diff
@@ -185,19 +242,19 @@ async function deploymentPrompt() {
 }
 
 /** Prompt the user to enter the ID of a study */
-async function studyIDPrompt() {
+// helper to check if the given study (input) is in firestore
+const validateStudyFirebase = async (input) => {
```

Comment:
I would keep this as a normal function to keep consistent with the other functions in the file. Generally I only use arrow functions if the function can be defined on a single line, but that's personal preference.
```suggestion
async function validateStudyFirebase(input) {
```